src/cephadm/cephadm

```diff
@@ -7836,10 +7843,6 @@ def _get_parser():
         '--reconfig',
         action='store_true',
         help='Reconfigure a previously deployed daemon')
-    parser_deploy.add_argument(
-        '--allow-ptrace',
-        action='store_true',
-        help='Allow SYS_PTRACE on daemon container')
```

Comment:
This removes an option (`--allow-ptrace`) without deprecating it. If we cherry-pick this to pacific, it will destabilize users who might have relied on this.

What if you changed this to `help=argparse.SUPPRESS` and printed a warning to the user instead?

Reply:
@ktdreyer do you suggest I should drop this commit entirely?

Reply:
I think it's ok to drop it in Ceph's Quincy release.

If you're cherry-picking this change to the `pacific` branch, it's a good idea to not cherry-pick this removal to that stable branch. For example, I think Red Hat has  existing documentation that refers to `--allow-ptrace` right?